docs/manage/CLI.md

```diff
@@ -190,6 +193,107 @@ The `logout` command ends an authenticated CLI session
 viam logout
 ```
 
+### module
+
+The `module` command allows to you to manage custom modules.
+This includes:
+
+* Creating a new custom modular resource on your local filesystem
+* Updating an existing module with new changes
+* Uploading a new module to the Viam Registry
+* Updating an existing module in the Viam Registry
+
+```sh {class="command-line" data-prompt="$"}
+viam module create --organization=<org name> --name <module-id>
+viam module update --organization=<org name> --name <module-id>
+viam module upload --organization=<org name> --name <module-id> --platform <platform> --version <version> --module meta.json <packaged-module.tar.gz>
+```
+
+Examples:
+
+```sh {class="command-line" data-prompt="$"}
+# create a new module named 'my-module' in organization 'abc':
+viam module create --organization='abc' --name 'my-module'
+
+# update the existing 'my-module' module with new changes to meta.json:
+viam module update --organization='abc' --name 'my-module'
+
+# upload a new or updated custom module 'my-module' to the Viam Registry:
+viam module upload --organization='abc' --platform "darwin/amd64" --version "1.0.0-rc1" --module meta.json packaged-module.tar.gz
+```
+
+#### Command options
+
+|        command option     |       description      | positional arguments
+| ----------- | ----------- | ----------- |
+| `create`    | create a new custom module on your local filesystem  | - |
+| `update`    | update an existing custom module on your local filesystem with recent changes to <file>meta.json</file> | - |
+| `upload`    | upload a new or existing custom module on your local filesystem to the Viam Registry (see [Using the upload command](#using-the-upload-command)) |
+| `help`      | return help      | - |
+
+##### Named arguments
+
+|        argument     |       description | applicable commands | required
+| ----------- | ----------- | ----------- | ----------- |
+| `--organization`      | organization name that the robot belongs to       |`create`, `update`, `upload`|true |
```

Comment:
Whoops I'll change this from `robot` to something else, but what is the scope of org ownership with module? Can you help me clarify how `module `commands make use of this flag's value?